src/components/Amount/Amount.tsx

```diff
@@ -153,6 +153,17 @@ const Percent = ({ value, autoColor, options, ...props }: PercentAmountProps) =>
   )
 }
 
+const Supply = ({ value, ...props }: AmountProps) => {
+  const {
+    number: { toSupply },
+  } = useLocaleFormatter({ fiatType: 'USD' })
```

Comment:
non-blocking comment, but this will change soon when multi currency is implemented, please be aware